.env

```diff
@@ -1,4 +1,9 @@
-# On running the 'build' commands this file is replaced by the files in `env-config/`
-# To ignore and keep this file we have a command in `postshrinkwrap` which assumes this file is not changing	
-# The command is `git update-index --assume-unchanged .env`
-# Source: https://wildlyinaccurate.com/git-ignore-changes-in-already-tracked-files/
\ No newline at end of file
+## NB only variables prefixed with SIMORGH_ are available on both the server and client
```

Comment:
Hey Lily - we don't want to update this file. Can you revert it and re-push?

Once you've done that, some combination of 

```
git update-index --no-assume-unchanged .env
git reset .env    
git update-index --assume-unchanged .env  
```

Should prevent it happening in future.